src/content/developers/tutorials/how-to-write-and-deploy-an-nft.md

```diff
@@ -0,0 +1,334 @@
+
+## Step 2: Create your app (and API key)
+
+Once you’ve created an Alchemy account, you can generate an API key by creating an app. This will allow us to make requests to the Ropsten test network. Check out [this guide](https://docs.alchemyapi.io/guides/choosing-a-network) if you’re curious to learn more about test networks.
+
+1. Navigate to the “Create App” page in your Alchemy Dashboard by hovering over “Apps” in the nav bar and clicking “Create App”
+
+![Create your app](https://cdn-images-1.medium.com/max/5200/0*XQGWtaqrAdJNm5xq.png)
+
+2. Name your app (we chose “My First NFT!”), offer a short description, select “Staging” for the Environment (used for your app bookkeeping), and choose “Ropsten” for your network.
+
+![Configure and publish your app](https://cdn-images-1.medium.com/max/5200/0*AFYHVgRTeBsEfTPR.png)
+
+3. Click “Create app” and that’s it! Your app should appear in the table below.
+
+## Step 3: Create an Ethereum account (address)
+
+We need an Ethereum account to send and receive transactions. For this tutorial, we’ll use Metamask, a virtual wallet in the browser used to manage your Ethereum account address. If you want to understand more about how transactions on Ethereum work, check out [this page](https://ethereum.org/en/developers/docs/transactions/) from the Ethereum foundation.
+
+You can download and create a Metamask account for free [here](https://metamask.io/download.html). When you are creating an account, or if you already have an account, make sure to switch over to the “Ropsten Test Network” in the upper right (so that we’re not dealing with real money).
+
+![Set Ropsten as your network](https://cdn-images-1.medium.com/max/2000/0*HFn3xEInPmKY6_al.png)
+
+Step 4: Add ether from a Faucet
+
+In order to deploy our smart contract to the test network, we’ll need some fake Eth. To get Eth you can go to the [Ropsten faucet ](https://faucet.ropsten.be/)and enter your Ropsten account address, then click “Send Ropsten Eth.” You should see Eth in your Metamask account soon after!
+
+## Step 5: Check your Balance
+
+To double check our balance is there, let’s make an [eth_getBalance](https://docs.alchemyapi.io/alchemy/documentation/alchemy-api-reference/json-rpc#eth_getbalance) request using [Alchemy’s composer tool](https://composer.alchemyapi.io?composer_state=%7B%22network%22%3A0%2C%22methodName%22%3A%22eth_getBalance%22%2C%22paramValues%22%3A%5B%22%22%2C%22latest%22%5D%7D). This will return the amount of Eth in our wallet. After you input your Metamask account address and click “Send Request”, you should see a response like this:
+
+    {"jsonrpc": "2.0", "id": 0, "result": "0xde0b6b3a7640000"}
+
+**NOTE: **This result is in wei not eth. Wei is used as the smallest denomination of ether. The conversion from wei to eth is: 1 eth = 1⁰¹⁸ wei. So if we convert 0xde0b6b3a7640000 to decimal we get 1*1⁰¹⁸ which equals 1 eth.
```

Comment:
- Instead of `1⁰¹⁸ wei` use `10<sup>18</sup> wei`
- Instead of `from wei to eth is: 1 eth...`
  use `from wei to ether is 1 ether...` or `from wei to ETH is 1 ETH...`